lib/builder/src/schema.json

```diff
@@ -45,11 +45,13 @@
     },
     "eslintCacheLocation": {
       "type": "string",
-      "description": "Path to the cache file or directory"
+      "description": "Path to the cache file or directory",
+      "default": ".angular/cache/.eslintcache"
     },
     "stylelintCacheLocation": {
       "type": "string",
-      "description": "Path to the cache file or directory"
+      "description": "Path to the cache file or directory",
+      "default": ".angular/cache/.eslintcache"
```

Comment:
```suggestion
      "default": ".angular/cache/.stylelintcache"
```

Reply:
I cannot make updates on my fork as it is delete - I suggest you create a develop branch on your repo and merge this PR into that